x-pack/filebeat/module/suricata/_meta/docs.asciidoc

```diff
@@ -0,0 +1,39 @@
+:modulename: suricata
+
+== suricata module
+
+This is the suricata module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+TODO: document with what versions of the software is this tested
```

Comment:
Should we fix the TODO in this file before merging? What I worry is that we forget about it before we release it.

Reply:
I agree we can't forget this. But testing compatibility will take a little while. I would track this as a task to be completed as part of the initial release. I wouldn't necessarily have the initial PR wait on this.

Reply:
What I would add here is just the version you tested yourself with and used for creating this. That is what we normally do for Filebeat / Metricbeat.

Reply:
++ on creating a meta issue to track necessary steps for first release. could you create one?

Reply:
Since I'm handing off the module to Andrew I've created an ad hoc Google Docs with a bit of background, some links, and some issues. I'll let him decide how to track the follow-up work.

But I did add a task in this handoff doc, so we do have it somewhere. I tagged you there, you should have received a GDoc comment notif :-)

Reply:
Ah right, I will add the current version I used, though. That's straightforward enough :-)